qkernel/src/kernel_def.rs

```diff
@@ -401,18 +401,17 @@ pub fn child_clone(userSp: u64) {
     CPULocal::SetKernelStack(currTask.GetKernelSp());
 
     currTask.AccountTaskEnter(SchedState::RunningApp);
-    let pt = currTask.GetPtRegs();
 
-    let kernelRsp = pt as *const _ as u64;
+    let pt_regs_top = currTask.GetPtRegs() as *const _ as u64;
     CPULocal::Myself().SetEnterAppTimestamp(TSC.Rdtsc());
     //currTask.mm.VcpuEnter();
     CPULocal::Myself().SetMode(VcpuMode::User);
     currTask.mm.HandleTlbShootdown();
-    debug!("entering child task: kernelSp/PtRegs @ {:x}", kernelRsp);
+    debug!("entering child task: kernelSp => PtRegs @ {:x}", pt_regs_top);
     #[cfg(target_arch = "x86_64")]
-    SyscallRet(kernelRsp);
+    SyscallRet(pt_regs_top);
```

Comment:
pt_regs top is a linux thing, and it's a very specific naming (ptrace registers). I don't think it's a good idea to rename the stack pointer to `pt_regs_top` here.

IMHO `pt_regs_top` is even more comfusing: what do you mean by "top" of a data structure? Data grows upwards.

Reply:
The top of the PtRegs structure:
`let pt_regs_top = currTask.GetPtRegs() as *const _ as u64;`

Reply:
`GetPtRegs()` gives you the bottom (base), not the "top".